README.md

```diff
@@ -3102,7 +3102,7 @@ Server: spray-can/1.3.3
 
 ## GET /api/engine/:version/stats
 
-This endpoint returns some basic statistics on the current state of the engine. At the moment that includes the number of running workflows and the number of active jobs. 
+This endpoint returns some basic statistics on the current state of the engine. At the moment that includes the number of running workflows and the number of jobs belonging to running workflows with their respective status. 
```

Comment:
Should this include the jobs in aborting workflows too?  Either folded into the one histogram or broken out into their own histogram?